### PR TITLE
Feature flat select

### PR DESCRIPTION
**Pull request by JPLachance**
- Add the flat select component to the styleguide
- Small tweaks to allow flat-select to be used for pagination
- Set the right padding for the actions container

### CODE REVIEW

File: scss/controls/flat-select.scss [hunk 1]
```diff
@@ -0,0 +1,86 @@
+.flat-select {
+  background-color: $body-background-color;
+  border-radius: 2px;
+
+  $flat-select-option-size: 26px;
```
Comment (marilynechan) [inline scss/controls/flat-select.scss]: Move this in `variables.scss`. I cannot find the mockups, but shouldn't this have the same height as buttons?

File: scss/controls/flat-select.scss [hunk 2]
```diff
@@ -0,0 +1,86 @@
+.flat-select {
+  background-color: $body-background-color;
```
Comment (marilynechan) [inline scss/controls/flat-select.scss]: Please use colors from the palette for new styles. `colors.scss` should disappear.

File: scss/utility/white-space.scss [hunk 3]
```diff
@@ -58,4 +58,20 @@ $spacing: 0.5rem;
   .ml#{$i} {
     margin-left: $i * $spacing;
   }
+
+  .bottom#{$i} {
```
Comment (marilynechan) [inline scss/utility/white-space.scss]: Maybe move this in `position.scss` and document them?

**Comment by marilynechan** [comment]
Other than a few comments/questions, ship it!

File: scss/controls/flat-select.scss (hunk 1 above)
Comment (JPLachance) [inline scss/controls/flat-select.scss]: That's a really good question. Mockups in Avocode (Cloud_v3, Pagination) suggest a 26 x 26 box. I'll ask Marjorie.

**Comment by pochretien** [comment]
nice one ! :shipit:

File: scss/controls/flat-select.scss (hunk 2 above)
Comment (JPLachance) [inline scss/controls/flat-select.scss]: Done!

File: scss/utility/white-space.scss (hunk 3 above)
Comment (JPLachance) [inline scss/utility/white-space.scss]: Removed, was a test..

**Pull request merged**